xla/finetune/adapter.py

```diff
@@ -55,6 +58,8 @@ def setup(
 ):
```

Comment:
nit on line 55, do we want the checkpoint dir to be consistent with the other finetuning scripts, using `checkpoints/stabilityai/stablelm-base-alpha-3b`

Reply:
I don't think so. we wanted to get rid of stablelm 3b as the default, but haven't done it because it would require rerunning the numbers for the existing tutorials.